src/Symfony/Component/TypeInfo/Tests/Type/ArrayShapeTypeTest.php

```diff
@@ -76,6 +76,11 @@ public function testAccepts()
 
         $this->assertTrue($type->accepts(['foo' => true]));
         $this->assertTrue($type->accepts(['foo' => true, 'bar' => 'string']));
+
+        $type = new ArrayShapeType(['foo' => ['type' => Type::bool()]], sealed: false);
+
+        $this->assertTrue($type->accepts(['foo' => true, 'other' => 'string']));
+        $this->assertTrue($type->accepts(['other' => 'string', 'foo' => true]));
```

Comment:
This test would cover the last fix.
```suggestion
        $this->assertTrue($type->accepts(['other' => 'string', 'foo' => true]));
        $this->assertFalse($type->accepts(['other' => 'string', 'foo' => 'foo']));
```